src/GitHub.UI.Reactive/Controls/TwoFactorInput.xaml

```diff
@@ -19,11 +20,11 @@
         </Style>
     </UserControl.Resources>
     <StackPanel Orientation="Horizontal" UseLayoutRounding="True" SnapsToDevicePixels="True">
-        <TextBox x:Name="one" />
-        <TextBox x:Name="two" />
-        <TextBox x:Name="three" />
-        <TextBox x:Name="four" />
-        <TextBox x:Name="five" />
-        <TextBox x:Name="six" Margin="0" />
+        <TextBox x:Name="one" AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberOneTextBox}" />
+        <TextBox x:Name="two" AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberTwoTextBox}" />
+        <TextBox x:Name="three" AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberThreeTextBox}" />
+        <TextBox x:Name="four" AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberFourTextBox}" />
+        <TextBox x:Name="five" AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberFiveTextBox}" />
+        <TextBox x:Name="six" Margin="0"  AutomationProperties.AutomationId="{x:Static automation:AutomationIDs.TwoFactorAuthenticatonNumberSixTextBox}" />
```

Comment:
That should be possible! I just grabbed all controls that looked testable and put an ID on it 😬